lib/ruby_home/config/characteristics.yml

```diff
@@ -30,6 +30,18 @@
     ValidValues:
       '0': Inactive
       '1': Active
+- :name: :active_identifier
+  :description: Active Identifier
+  :uuid: 000000E7-0000-1000-8000-0026BB765291
+  :format: uint32
+  :unit: nil
```

Comment:
I assumed that this should be `nil` for all of the characteristics I added in this PR, I didn't see anything that would map to this field.

Reply:
Yes this is correct. They should all be `nil`